src/udev/udev-watch.c

```diff
@@ -44,50 +44,39 @@ int device_new_from_watch_handle_at(sd_device **ret, int dirfd, int wd) {
 }
 
 int udev_watch_restore(int inotify_fd) {
-        _cleanup_closedir_ DIR *dir = NULL;
+        _cleanup_(rm_rf_physicalp) const char *old = "/run/udev/watch.old";
```

Comment:
Hmm, _physical is not needed here? Plain rm_rf will do and is actually safer?

Reply:
OK. `REMOVE_PHYSICAL` is dropped and renamed to rm_rf_safe().